ui/shared/BugFiler.jsx

```diff
@@ -536,6 +543,35 @@ export class BugFilerClass extends React.Component {
     });
   };
 
+  checkForSecurityIssue() {
+    if (this.state.isSecurity) {
+      return;
+    }
+
+    const inputToCheck = `${this.state.summary}\n${this.state.comment}`;
+
```

Comment:
You could unpack the state values: `const { summary, comment, isSecurity } = this.state;`